packit/cli/generate.py

```diff
@@ -74,7 +73,15 @@ def generate(path_or_url, force):
     )
 
 
-def generate_config(write_to_file=False, template_data=None, config_file_name=None):
+def get_existing_config(path):
+    # find name of config file if already exists
+    for existing_config_file in CONFIG_FILE_NAMES:
+        if (Path(path) / existing_config_file).is_file():
+            return existing_config_file
+    return ""
```

Comment:
Maybe a `None` would be better?
```suggestion
    return None
```

Could you also add the type annotation, please?
```python
def get_existing_config(path:str) -> Optional[str]:
```